TripKit/Classes/model/API/AlertAPIModels.swift

```diff
@@ -61,8 +62,42 @@ extension API {
     public let operators: [String]?
     public let serviceTripIDs: [String]?
     public let stopCodes: [String]?
-    public let routeIDs: [String]?
+    public let routes: [API.Route]?
+    public let transportType: TransportType?
+  }
+  
+  public struct Route: Codable {
+    public let id: String
+    public let name: String?
+    public let number: String?
+    public let modeInfo: ModeInfo
+    
+    /// This color applies to an individual service.
+    public var color: UIColor? { return modeInfo.color }
   }
+  
+  /// This is a `Codable` representation of the transportType JSON. All routes affected
+  /// by the same alert will have the same transportType.
+  public struct TransportType: Codable {
```

Comment:
What is `TransportType`? I don't want to introduce a new concept if we already have `ModeInfo` and "mode identifiers"?